src/extension/features/toolkit-reports/index.js

```diff
@@ -75,8 +75,8 @@ export class ToolkitReports extends Feature {
     $(YNAB_NAVLINK_SELECTOR).removeClass('active');
     $(YNAB_NAVACCOUNT_SELECTOR).removeClass('is-selected');
     $(TOOLKIT_REPORTS_NAVLINK_SELECTOR).addClass('active');
-    $(`${YNAB_NAVLINK_SELECTOR}, ${YNAB_NAVACCOUNT_SELECTOR}`).on('click', () =>
-      this._removeToolkitReports()
+    $(`${YNAB_NAVLINK_SELECTOR}, ${YNAB_NAVACCOUNT_SELECTOR}`).on('click', (e) =>
+      this._removeToolkitReports(e)
     );
```

Comment:
I think we need to do 

```ts
$(`${YNAB_NAVLINK_SELECTOR}, ${YNAB_NAVACCOUNT_SELECTOR}`).on('click', this._removeToolkitReports);
``` 

or we might add an event listener every time `_updateNavigation` runs?

Reply:
This will attach multiple listeners too. I checked and it seem to be a 'problem' only if user repeatedly clicks on 'Toolkit reports' while they are already open. But anyway I updated, now there will be only one listener at a time